packages/paste-design-tokens/tokens/global/border-color.yml

```diff
@@ -8,24 +8,67 @@ props:
   color-border-dark:
     value: "{!palette-gray-60}"
     comment: Dark border color
+    uicontrol_contrast_pairing:
```

Comment:
How did you come up with these pairings and how should we review them? A little confused on how to provide helpful feedback here.

Reply:
Combination of kinda knowing what the system should do, whats used in the components currently, and best guess.

The one I have commented out are one that I think _should_ work but are currently failing too.

It's not definitive, but we can iterate on it. _Especially_ when we start thinking more about different background cases, as we really only cater for "on background body", they'll evolve

Reply:
Just making sure I understand this correctly. This is basically saying that this border token should have a passing contrast ratio score when used on certain background color tokens. Right?

For the ones that _should_ work, do you have any insight on why they're not working?

If my understanding is correct, this feels like a good first pass. I'm sure these will continue to evolve and need to be iterated on.

Reply:
You are spot on. `color-border-dark` has ui control contrast ratio requirements against the following colors.

> For the ones that should work, do you have any insight on why they're not working?

We picked the wrong colors? 🤷🏼